scripts/validate-env.js

```
@@ -0,0 +1,249 @@
+
+/**
+ * Environment Variables Validation Script
+ * 
+ * This script validates that all required Auth0 environment variables
+ * are properly configured for both frontend and backend.
+ * 
+ * Usage:
+ *   node scripts/validate-env.js
+ */
+
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)